dvc/config.py

```diff
@@ -468,6 +468,19 @@ def validate(data):
             raise ConfigError(str(exc)) from None
 
 
+def _format_config(config, parent=None):
+    for section, options in config.items():
+        for option, value in options.items():
+            key = f"{section}.{option}"
+            if parent is not None:
+                key = f"{parent}.{key}"
+
+            if isinstance(value, dict):
+                yield from _format_config({option: value}, parent=section)
```

Comment:
Seems like we could simply use `flatten` from `dvc.utils.flatten` and avoid walking the config in general.

Reply:
Wasn't aware of that, thanks!